tests/snippets/name_import.py

```diff
@@ -0,0 +1,2 @@
+print(__name__)
+import name
```

Comment:
Huh? Is this possible? Are you sure this is correct?

Reply:
Hi Windel, 
My intention here was to demonstrate that __name__ is set on import.
I have renamed these snippets and added some assertions to (hopefully) make this clearer.